dolphinscheduler-task-plugin/dolphinscheduler-task-sqoop/src/main/java/org/apache/dolphinscheduler/plugin/task/sqoop/parameter/SqoopParameters.java

```diff
@@ -223,13 +223,13 @@ public ResourceParametersHelper getResources() {
         }
 
         SourceMysqlParameter sourceMysqlParameter = JSONUtils.parseObject(this.getSourceParams(), SourceMysqlParameter.class);
-        if (sourceMysqlParameter.getSrcDatasource() != 0) {
-            resources.put(ResourceType.DATASOURCE, sourceMysqlParameter.getSrcDatasource());
+        if (sourceMysqlParameter.getSrcDataSource() != 0) {
+            resources.put(ResourceType.DATASOURCE, sourceMysqlParameter.getSrcDataSource());
         }
 
         TargetMysqlParameter targetMysqlParameter = JSONUtils.parseObject(this.getTargetParams(), TargetMysqlParameter.class);
-        if (targetMysqlParameter.getTargetDatasource() != 0) {
-            resources.put(ResourceType.DATASOURCE, targetMysqlParameter.getTargetDatasource());
+        if (targetMysqlParameter.getTargetDataSource() != 0) {
+            resources.put(ResourceType.DATASOURCE, targetMysqlParameter.getTargetDataSource());
```

Comment:
You can't do this change directly, this will bring an upgrade issue, the parameter has been store at database, user need to refresh their database.